PWGHF/D2H/Tasks/taskLcToK0sP.cxx

```diff
@@ -60,6 +60,8 @@ struct HfTaskLcToK0sP {
       registry.add("hPtGenSig", "cascade candidates (MC);#it{p}_{T}^{gen.} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}});
       registry.add("hCPARecSig", "cascade candidates (matched);cosine of pointing angle;entries", {HistType::kTH1F, {{110, -1.1, 1.1}}});
       registry.add("hCPARecBg", "cascade candidates (unmatched);cosine of pointing angle;entries", {HistType::kTH1F, {{110, -1.1, 1.1}}});
+      registry.add("hV0CPARecSig", "cascade candidates (matched);v0 cosine of pointing angle;entries", {HistType::kTH1F, {{100, 0.98, 1.0001}}});
+      registry.add("hV0CPARecBg", "cascade candidates (unmatched);v0 cosine of pointing angle;entries", {HistType::kTH1F, {{100, 0.98, 1.0001}}});
```

Comment:
If you make the data histograms 2D, you should also make these MC ones 2D no?

Reply:
OK, done